documentation/providers/axfrddns.md

```diff
@@ -132,6 +134,16 @@ the following error message:
 Please consider adding default `nameservers` or an explicit `master` in `creds.json`.
 ```
 
+### Buggy DNS servers regarding CNAME updates
+
+When modifying a CNAME record, or when replacing an A record by a
+CNAME one in a single batched DDNS update, some DNS servers
+(e.g. Knot) will incorrectly reject the update. For this particular
+case, you might set the option `buggy-cname = "yes"` in `creds.json`.
+The changes will then be splitted in two DDNS updates, applied
```

Comment:
s/splitted/split/    

(English is a weird language)

Reply:
Done, thanks.